AppController/lib/haproxy.rb

```diff
@@ -112,7 +122,13 @@ def self.restart()
   end
 
   def self.reload()
+    (PORTS + (10000..10020).to_a).each { |port|
+      Djinn.log_run("iptables -I INPUT -p tcp --dport #{port} --tcp-flags SYN,PSH,ACK,FIN,RST ACK -m u32 \! --u32 '0>>22&0x3C@12>>26&0x3C@0&0x0=0:0' -j DROP")
```

Comment:
In a blog you referenced, dropping only SYN packages is suggested. Why do you think we need to block packages with other flags?